docs/explanation/workflow-and-permissions.md

```diff
@@ -62,10 +67,11 @@ flowchart TD
         C -- Return prior to full review --> E
         D -- Consolidation --> E
         E -- Updates and responds to questions --> B
-        D -- All files approved ----> F(RELEASED)
+        D -- All files approved ----> F(APPROVED)
         D -- Not approved ----> G(REJECTED)
-        A --> H(WITHDRAWN)
-        E --> H
+        F(APPROVED) -- Files uploaded ----> H(RELEASED)
+        A --> I(WITHDRAWN)
+        E --> I
     end
```

Comment:
this change looks good. whilst looking at it I realise that `RETURNED` currently only leads to `WITHDRAWN` but I can do that nit separately

Reply:
Returned also leads back to submitted (with "updates and responds to questions"  - i.e. resubmitting). I don't think there's anywhere else it should go.  That's consistent with the auto-generated [state diagram](https://async-release-docs.airlock.pages.dev/reference/request-states/)

Reply:
oh sorry, I misread it!! as you were